.github/workflows/increment-patch.yml

```diff
@@ -0,0 +1,40 @@
+name: Increment Patch
+
+on:
```

Comment:
Same here. Can you add https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows as a link of a quick explanation of the events